index.bs

```diff
@@ -1609,12 +1609,11 @@ of the SuperSpeed Endpoint Companion descriptor.
 
 # Integrations # {#integrations}
 
-## Feature Policy ## {#feature-policy}
```

Comment:
Is it possible to keep the old anchor around? It looks like the CI build is failing because there are still references to it from MDN.

Reply:
Yes, we should be able to do that, but might have to stray from markdown syntax. Bikeshed supports an `oldids` attribute on the header element that should keep the xrefs working.